Sources/ArcGISToolkit/Components/FloatingPanel/FloatingPanel.swift

```diff
@@ -73,7 +73,7 @@ struct FloatingPanel<Content>: View where Content: View {
             // Set frame width to infinity to prevent horizontal shrink on dismissal.
             .frame(maxWidth: .infinity)
 #if os(visionOS)
-            .background(.regularMaterial)
+            .glassBackgroundEffect()
             .compositingGroup()
```

Comment:
Is this necessary? Methinks not, but I'm not entirely certain.

Reply:
Yep if we remove `compositingGroup` and `clipShape` it looks like this:
![Screenshot 2025-01-31 at 4 38 01 PM](https://github.com/user-attachments/assets/4970901d-6b12-4e37-a55d-af19b0ed6370)